.github/workflows/greet_first_time_contributor.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - uses: zephyrproject-rtos/action-first-interaction@7e6446f8439d8b4399169880c36a3a12b5747699 # v1.1.1-zephyr-5
+      - uses: zephyrproject-rtos/action-first-interaction@1d8459ca65b335265f1285568221e229d45a995e # v1.1.1-zephyr-5
```

Comment:
This is wrong -- will look into it

Reply:
you probably need to change the versioning scheme

Reply:
Yep, exactly